### PR TITLE
chore: add target for esbuild lowering testing

### PR DESCRIPTION
**Pull request by IWANABETHATGUY**

### CODE REVIEW

**Comment by IWANABETHATGUY** [comment]
* **#2672** <a href="https://app.graphite.dev/github/pr/rolldown/rolldown/2672?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @IWANABETHATGUY and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: crates/rolldown_common/src/inner_bundler_options/types/target.rs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+#[cfg(feature = "deserialize_bundler_options")]
+use schemars::JsonSchema;
+#[cfg(feature = "deserialize_bundler_options")]
+use serde::Deserialize;
+
+#[derive(Debug, Clone, Copy, Default)]
+#[cfg_attr(feature = "deserialize_bundler_options", derive(Deserialize, JsonSchema))]
+#[cfg_attr(feature = "deserialize_bundler_options", serde(rename_all = "camelCase"))]
+pub enum ESTarget {
+  ES5,
```
Comment (hyf0) [inline crates/rolldown_common/src/inner_bundler_options/types/target.rs]: NIT: `EsTarget`, `Es5` are more rust-style names. The std follows this style.

https://rust-lang.github.io/api-guidelines/naming.html#:~:text=In%20UpperCamelCase%2C%20acronyms%20and%20contractions%20of%20compound%20words%20count%20as%20one%20word%3A%20use%20Uuid%20rather%20than%20UUID%2C%20Usize%20rather%20than%20USize%20or%20Stdin%20rather%20than%20StdIn.

**Review by hyf0 — APPROVED** [review]

File: crates/rolldown_common/src/inner_bundler_options/types/target.rs (hunk 1 above)
Comment (IWANABETHATGUY) [reply crates/rolldown_common/src/inner_bundler_options/types/target.rs]: I just copy it from oxc https://github.com/oxc-project/oxc/blob/b4258ee58e0a94773ccf85511530153025a58f4a/crates/oxc_transformer/src/options/es_target.rs#L7-L22

File: crates/rolldown_testing/_config.schema.json [hunk 2]
```diff
@@ -462,6 +472,23 @@
         }
       }
     },
+    "ESTarget": {
+      "type": "string",
+      "enum": [
+        "eS5",
```
Comment (hyf0) [inline crates/rolldown_testing/_config.schema.json]: LoL. Looks like that's one of the reason for https://github.com/rolldown/rolldown/pull/2672/files#r1835472534

**Pull request merged**